optimum/exporters/openvino/model_patcher.py

```diff
@@ -951,15 +951,103 @@ def __exit__(self, exc_type, exc_value, traceback):
                 block.attention.forward = block.attention._orig_forward
 
 
+# Adapted from Phi3Attention.forward
+def _phi3_self_attn_sdpa_forward(
+    self,
+    hidden_states: torch.Tensor,
+    attention_mask: Optional[torch.Tensor] = None,
+    position_ids: Optional[torch.LongTensor] = None,
+    past_key_value: Optional[Tuple[torch.Tensor]] = None,
+    output_attentions: bool = False,
+    use_cache: bool = False,
+) -> Tuple[torch.Tensor, Optional[torch.Tensor], Optional[Tuple[torch.Tensor]]]:
+    if output_attentions:
+        return self._orig_forward(
+            hidden_states=hidden_states,
+            attention_mask=attention_mask,
+            position_ids=position_ids,
+            past_key_value=past_key_value,
+            output_attentions=output_attentions,
+            use_cache=use_cache,
+        )
+
+    from transformers.models.llama.modeling_llama import apply_rotary_pos_emb, repeat_kv
```

Comment:
even if currently equivalent shouldn't be replace it with 
```suggestion
    from transformers.models.phi3.modeling_phi3 import apply_rotary_pos_emb, repeat_kv
```
to avoid any issue resulting from potential refactorization

Reply:
I believe it will breaks with current release, because phi3 code in transformers is not released yet... Can we add some TODO to change this in future?

Reply:
yes works for me, thanks @eaidova !